irssi/README.md

```diff
@@ -0,0 +1,29 @@
+Dockerfile-Fedora-irssi
+=======================
+
+Fedora Dockerfile for [irssi](https://irssi.org/).
+
+To Run the Container:
+---------------------
+
+```
+# docker run -it fedora/irssi
+```
+
+This will start the irssi client
+
+To Build:
+---------
+
+Copy the sources down and perform Build
+
+```
+# docker build -t <username>/irssi .
+```
+
+To Run:
+-------
+
+```
+# docker run -it <username>/irssi
```

Comment:
OK.  Makes sense.  If you are targeting atomic users, should you leverage "atomic run" instead?